registration/src/main/java/com/kyriba/registration/domain/TicketStatus.java

```diff
@@ -0,0 +1,24 @@
+package com.kyriba.registration.domain;
+
+public enum TicketStatus
```

Comment:
TicketStatus in attendee microservice?

Reply:
extract Ticket related logic to Ticket service?

Reply:
it sounds logical, if this functionality was not planned to be implemented in registration service

Reply:
according our Domain model diagram Ticket is a part of Registration service, so I'll leave this functionality in this service, but place endpoints in TicketController.